spec/lib/rabbit_feed/consumer_connection_spec.rb

```diff
@@ -29,6 +29,18 @@ module RabbitFeed
         expect(bunny_channel).to receive(:prefetch).with(1)
         subject
       end
+
+      context 'when RabbitFeed.route_prefix_extension is set' do
```

Comment:
`RabbitFeed.route_prefix_extension` no longer exists - could change to `when a route_prefix_extension is set`